rfcs/formats/communication/relay.md

```diff
@@ -12,3 +12,51 @@
 
 ## Summary
 
+
+## NetworkQuery
+
+This message is created by client application to encapsulate the query for the relay.
```

Comment:
We should say a bit more about the purpose of this structure. How about: `Whenever a client application needs to query a foreign network for a ledger state view (add link around this word to models/ledger/views.md), it must wrap the query, or a view request (link to request.md) comprising of a view address (link to addressing.md), a verification policy (link to the verification policy spec), and other metadata, into a DLT-neutral structure as follows for the relays (link to relay.md in models) that will accept and route the request:`?